scripts/secnetperf.ps1

```diff
@@ -98,7 +98,14 @@ $useXDP = ($io -eq "xdp" -or $io -eq "qtip")
 # Set up the connection to the peer over remote powershell.
 Write-Host "Connecting to $RemoteName"
 if ($isWindows) {
-    $Session = New-PSSession -ComputerName $RemoteName -ConfigurationName PowerShell.7
+    if ($os -eq "windows-2025") {
```

Comment:
Should we simply just _always_ do this? If it works for 2022, I think I'd rather have the simpler script.

Reply:
lets try and see if this works for WS 2022

Reply:
it looks like it works. Lab failures are unrelated, currently doing some tweaking to diagnose poor performance.